server/sonar-web/src/main/js/app/components/extensions/OrganizationPageExtension.js

```diff
@@ -27,6 +27,7 @@ import type { Organization } from '../../../store/organizations/duck';
 
 type Props = {
   organization: Organization,
+  location: { query: { page: string } },
```

Comment:
you can simply write `location: {}`, because not every organization extension will have a page parameter